src/transport/SessionHandle.cpp

```diff
@@ -23,12 +23,33 @@ namespace chip {
 
 using namespace Transport;
 
+using AuthMode          = Access::AuthMode;
 using SubjectDescriptor = Access::SubjectDescriptor;
 
 SubjectDescriptor SessionHandle::GetSubjectDescriptor() const
 {
-    SubjectDescriptor subjectDescriptor = { .fabricIndex = mFabric };
-    // TODO: fill subject descriptor with proper fields
+    SubjectDescriptor subjectDescriptor;
+    if (IsSecure())
+    {
+        if (IsOperationalNodeId(mPeerNodeId))
+        {
+            subjectDescriptor.authMode    = AuthMode::kCase;
+            subjectDescriptor.subject     = mPeerNodeId;
+            subjectDescriptor.fabricIndex = mFabric;
+            // TODO(#10243): add CATs
+        }
+        else if (IsPAKEKeyId(mPeerNodeId))
```

Comment:
So I was trying to find where we create SessionHandles with mPeerNodeId set to something in that range, and I haven't found it yet.  Could you point me to that code, please?

Reply:
It isn't. This won't work until it is.

Reply:
OK.  Can you please leave in a TODO comment for that or something, ideally pointing to an existing issue tracking it?

Reply:
Well it's not a TODO for this code, but of course it's already being tracked: issue #10242